sdk/identity/identity-brokered-auth/src/index.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzurePluginContext, NativeBrokerPluginOptions } from "../../identity/src/plugins/provider";
+import { IdentityPlugin } from "@azure/identity";
+import { NativeBrokerPlugin } from "@azure/msal-node-extensions";
+
+/**
+ * A plugin that provides WAM Integration for `@azure/identity`
+ * credentials. The plugin API is compatible with `@azure/identity` versions
+ * 2.0.0 and later. Load this plugin using the `useIdentityPlugin`
+ * function, imported from `@azure/identity`.
+ *
+ * Example:
+ *
+ * ```javascript
+ * import { useIdentityPlugin, DeviceCodeCredential } from "@azure/identity";
+ * import { nativeBrokerPlugin } from "@azure/identity-cache-persistence";
+ *
+ * // Load the plugin
+ * useIdentityPlugin(nativeBrokerPlugin);
+ *
+ * const credential = new DeviceCodeCredential({
+ *   enableMsaPassthrough: true
+ * });
+ * ```
+ */
+
+export function createNativeBrokerPlugin(options: NativeBrokerPluginOptions): IdentityPlugin {
+  return async function nativeBrokerPlugin({ nativeBrokerPluginControl }: AzurePluginContext) {
+    let nativeBrokerPlugin = new NativeBrokerPlugin();
+    nativeBrokerPlugin.isBrokerAvailable = true;
```

Comment:
Please do not set this, this is internal to msal-node-extensions. I'll make a note to make this private there